script/test.sh

```diff
@@ -9,62 +9,114 @@
 #
 # It outputs test and coverage reports in a way that Jenkins can understand,
 # with test results in JUnit format and test coverage in Cobertura format.
-# The reports are saved to build/$SUBDIR/{test-reports,coverage-reports}/*.xml 
+# The reports are saved to build/component/{test-reports,coverage-reports}/*.xml
 #
 set -e
 set -o pipefail
 export PATH="${GOPATH}/bin:${PATH}"
 
-PACKAGES="$(go list ./... | grep -v /vendor/)"
-SUBDIRS="cmd"
+if [ $# -ne 1 ]; then
+    echo "usage: $0 [test suite]"
```

Comment:
`[]` usually indicates optional arguments, but the test suite appears to be a required argument. And `test suite` should be one word to indicate that it's a single arg.

How about:
```
echo "usage: $0 test_suite"
```